Force fallback to traditional executable detection.

This is to fix crbug/591436, which is a blocker on using clang for
simple-chrome.

diff --git a/lib/Support/Unix/Path.inc b/lib/Support/Unix/Path.inc
index d85c37a..dc15d54 100644
--- a/lib/Support/Unix/Path.inc
+++ b/lib/Support/Unix/Path.inc
@@ -144,11 +144,11 @@ std::string getMainExecutable(const char *argv0, void *MainAddr) {
   if (getprogpath(exe_path, argv0) != NULL)
     return exe_path;
 #elif defined(__linux__) || defined(__CYGWIN__)
   char exe_path[MAXPATHLEN];
   StringRef aPath("/proc/self/exe");
-  if (sys::fs::exists(aPath)) {
+  if (false && sys::fs::exists(aPath)) {
       // /proc is not always mounted under Linux (chroot for example).
       ssize_t len = readlink(aPath.str().c_str(), exe_path, sizeof(exe_path));
       if (len >= 0)
           return std::string(exe_path, len);
   } else {
